src/lib/RawMapData.ts

```diff
@@ -45,23 +47,18 @@ export interface RawMapLayerMetaData {
     active?: boolean;
 }
 
-export enum RawMapLayerType {
-    Floor = "floor",
-    Segment = "segment",
-    Wall = "wall",
-}
+export type RawMapLayerType = "floor" | "segment" | "wall"
 
-export enum RawMapEntityType {
-    ChargerLocation = "charger_location",
-    RobotPosition = "robot_position",
-    GoToTarget = "go_to_target",
-    Path = "path",
-    PredictedPath = "predicted_path",
-    VirtualWall = "virtual_wall",
-    NoGoArea = "no_go_area",
-    NoMopArea = "no_mop_area",
-    ActiveZone = "active_zone",
-}
+export type RawMapEntityType =
+    | "charger_location"
```

Comment:
Replaced enum with a simple union, as the rest of the code doesn't use enums anyway (e.g. `RawMapEntityType.ChargerLocation`).